src/Fabulous.Avalonia/Fabulous.Avalonia.fsproj

```diff
@@ -222,7 +222,7 @@
       FSharp.Core is fixed to a specific version that is not necessarily the latest one.
       This version will be used as the lower bound in the NuGet package
     -->
-    <PackageReference Include="Fabulous" VersionOverride="[2.3.0, 2.4.0)" />

```

Comment:
Fyi: you don't actually need to update this version override since it's already a range.
NuGet will allow users to choose the Fabulous version they prefer.

I had to update it in Fabulous.MauiControls because we did a breaking change `[2.2.0, 2.3.0)` (supports version 2.2.0 to 2.3.0 not included) to `[2.3.0, 2.4.0)`